Firestore/Example/Tests/Local/FSTRemoteDocumentCacheTests.mm

```diff
@@ -92,29 +125,33 @@ - (void)testSetAndReadSeveralDocuments {
   if (!self.remoteDocumentCache) return;
 
   self.persistence.run("testSetAndReadSeveralDocuments", [=]() {
```

Comment:
Ultranit: the empty argument list in the lambda can be omitted (here and below).